docs/config/io_helidon_integrations_neo4j.adoc

```diff
@@ -0,0 +1,57 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+// MANUALLY CREATED DOC
+
+:description: Configuration of io_helidon_integrations_neo4j
+:keywords: helidon, config, io_helidon_integrations_neo4j.adoc
+:basic-table-intro: The table below lists the configuration keys that configure io_helidon_integrations_neo4j.adoc
+
+= Neo4j Configuration
+
+// tag::config[]
+
+
+MicroProfile configuration options:
+[cols="3,3,2,5"]
```

Comment:
Rerun the doc generator with Tomas's latest changes?

Reply:
as commented above, no changes in Neo4j module code for now.